src/secret_sharing/utils.rs

```diff
@@ -32,6 +32,7 @@ pub fn compute_sha256_hash(input: &[u8]) -> Vec<u8> {
     hasher.finalize().to_vec()
 }
 
+#[allow(dead_code)]
 pub fn prg_seed_expansion(rand: &[u8]) -> Option<[u8; 32]> {
     if rand.len() != 16 { //expecting only 128 bit keys
```

Comment:
Why do we expect only 128-bit keys?

Reply:
I think we should eliminate this function altogether. It's being invoked by the ADSS construction, that we are not using anymore. And if we were to retain it, we could perhaps extract 256 bits directly from its invocation of the random oracle.

Reply:
@johnalanwoods Added a commit to remove the dead code mentioned above.